### DISCUSSION TITLE
Github intro guides

### DISCUSSION THREAD

**Discussion by Maurphal** [sdy60/syllabus#91] [Show and tell]
[Introduction to GitHub](https://github.com/skills/introduction-to-github)

**Comment by epidrome** [comment]
@Maurphal πήγα να καρφίτσωσω την νέα σου ανάρτηση για τους πίνακες και κατά λάθος την έσβησα ενώ η διεπαφή ούτε καν μου ζήτησε επιβεβαίωση αν μπορείς να το αναρτήσεις πάλι ευχαριστώ και συγγνώμη!

**Reply by Maurphal** [reply]
ok np. Done.

**Comment by Maurphal** [comment]
https://www.freecodecamp.org/news/guide-to-git-github-for-beginners-and-experienced-devs/

Βρήκα ακόμα αυτόν τον οδηγό. Φαίνεται καλός.
